docs/import.md

```diff
@@ -1,50 +1,89 @@
+Importing a model requires three steps:
 
- - LlamaForCausalLM
- - MistralForCausalLM
- - GemmaForCausalLM
+  1. Write a Modelfile
+  2. Create your model with `ollama create my-model`
+  3. Test your model with `ollama run my-model`
+
+
+#### Writing a Modelfile
+
+Create a Modelfile using a text editor which includes a `FROM` line which points to the directory with the model weights.
 
 ```dockerfile
 FROM /path/to/safetensors/directory
 ```
 
-For architectures not directly convertable by Ollama, see llama.cpp's [guide](https://github.com/ggerganov/llama.cpp/blob/master/README.md#prepare-and-quantize) on conversion. After conversion, see [Import GGUF](#import-gguf).
+If you create the Modelfile in the same directory as the weights, you can use the line `FROM .`.
 
-## Automatic Quantization
+You can include other settings in your Modelfile such as parameters for the model, the desired chat template, the system prompt, and any license information. Refer to the [Modelfile documentation](https://github.com/ollama/ollama/blob/main/docs/modelfile.md) for more information about Modelfile commands.
 
-> [!NOTE]
-> Automatic quantization requires v0.1.35 or higher.
+#### Create your model with `ollama create`
 
-Ollama is capable of quantizing FP16 or FP32 models to any of the supported quantizations with the `-q/--quantize` flag in `ollama create`.
+After you have created the Modelfile, use the command:
 
-```dockerfile
-FROM /path/to/my/gemma/f16/model
+```bash
+$ ollama create -f Modelfile <model name>
 ```
 
-```shell
-$ ollama create -q Q4_K_M mymodel
-transferring model data
-quantizing F16 model to Q4_K_M
-creating new layer sha256:735e246cc1abfd06e9cdcf95504d6789a6cd1ad7577108a70d9902fef503c1bd
-creating new layer sha256:0853f0ad24e5865173bbf9ffcc7b0f5d56b66fd690ab1009867e45e7d2c4db0f
-writing manifest
-success
+If the model architecture of your weights is supported, Ollama will take a few moments to import the weights. It will also attempt to find the correct _chat template_ inside the model's configuration data. You can override the template using the `TEMPLATE` command in your Modelfile.
+
+#### Test your model with `ollama run`
+
+Once you have created a model, you can run it using the command:
+
+```bash
+$ ollama run <model name>
 ```
 
+This will load the newly created model into memory and you can test to make certain that it's working correctly. You may have to make changes to your Modelfile and run the `ollama create` command again in order to get everything correct.
+
+After it is working, if you have created an account on [ollama.com](https://ollama.com), you can push your model to [ollama.com](https://ollama.com) to share it with other people. Use the name `<user name>/<model name>` (e.g. `jmorganca/my-model`) when you are creating the model, or use the `ollama cp` command to name the model with your user name at the beginning of the model name. You can then push the model using the command:
```

Comment:
This is great – should we put it under a `Push your model to ollama.com` header?